tests/integration/publish/test_command_integ.py

```diff
@@ -32,67 +37,65 @@ def setUp(self):
     def tearDown(self):
         super(TestPublishExistingApp, self).tearDown()
         # Delete application for each test
-        self.sar_client.delete_application(ApplicationId=self.application_id)
+        # self.sar_client.delete_application(ApplicationId=self.application_id)
```

Comment:
should this be uncommented?

Reply:
Yes, I added that to debug, I'll fix this.